Common/Systems/ModPlayers/ClassModPlayer.cs

```diff
@@ -29,12 +31,16 @@ public override void ModifyStartingInventory(IReadOnlyDictionary<string, List<It
 		item.TurnToAir();
 		vanillaItems.Add(item);
 
-		item = new Item();
-		item.SetDefaults(ItemID.CopperPickaxe);
+		item = new Item(ModContent.ItemType<MagicWoodenSword>());
+		vanillaItems.Add(item);
```

Comment:
I think this would be something like
var sword = new Sword();
sword.Rarity = magic;
sword.Roll();
vanillaItems.Add(sword);

Something like that, might not even need the roll. It might do it automatically.

This is probably not 100% accurate, been a sec since I looked at this codebase